ReactiveLists.podspec

```diff
@@ -0,0 +1,18 @@
+Pod::Spec.new do |s|
+  s.name             = "ReactiveLists"
+  s.version          = "0.0.1"
+  s.summary          = "React-like API for UITableView and UICollectionView"
+  s.description      = <<-DESC
+                        React-like API for UITableView and UICollectionView 
+                        DESC
+  s.homepage         = "https://github.com/plangrid/ReactiveLists"
+  s.license          = { :type => "MIT", :file => "LICENSE.md" }
+  s.author           = "PlanGrid"
+  s.documentation_url = "https://plangrid.github.io/ReactiveLists"
+  s.social_media_url = "https://medium.com/plangrid-technology"
```

Comment:
should this be twitter?

Reply:
Hm..not sure.. as there's nothing tech related being shared via twitter (all business)